wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -144,8 +140,8 @@ def parse_article(self, response):
         else:
             err_link = href if href else ''.join([response.url, ' (referer)'])
```

Comment:
how can we be here and not have an href?

Reply:
Some of the articles don't have any pdf attached to them. It is mainly avoided by the addition of `filternumbits=apply` in the first search urls, but somehow, I still have some cases where it happens